libs/plugin-elliptic/tests/ed25519.spec.ts

```diff
@@ -135,44 +147,114 @@ describe('ed25519 - EDDSA', () => {
+        referenceSignature: Buffer.from('0aab4c900501b3e24d7cdf4663326a3a87df5e4843b2cbdb67cbf6e460fec350aa5371b1508f9f4528ecea23c436d94b5e8fcd4f681e30a6ac00a9704a188a03', 'hex')
+      },
+      {
+        name: 'https://github.com/transmute-industries/vc.js reference',
+        jwk: {
+          crv: 'Ed25519',
+          d: base64url.encode(Buffer.from('9b937b81322d816cfab9d5a3baacc9b2a5febe4b149f126b3630f93a29527017', 'hex')),
+          x: base64url.encode(Buffer.from('095f9a1a595dde755d82786864ad03dfa5a4fbd68832566364e2b65e13cc9e44', 'hex')),
+          kty: 'OKP'
+        },
+        referenceMessage: Buffer.from('65794a68624763694f694a465a45525451534973496d49324e4349365a6d467363325573496d4e79615851694f6c7369596a5930496c31392e7971500a44d02ed0678e0f670e0789061ad147f659bca6390b82464cd4e439358e261497212183c4f54d354f55fb7929dbf4f4e2c5a6a232c5c52b1cc3e45ff5', 'hex'),
+        referenceSignature: Buffer.from('b950aaa5e9eed45e2d9c4034840614d5a6f1fa4e970b5274f0f90a478fc69f120bf59eed5ee98fd87724925219e4051dba763afafee1e948de0623c54544c10f', 'hex')
+      }
+    ]
+    for (let reference in references) {
+      console.log(references[reference].name);
+      await validate(references[reference]);
+    }
   });
 
   it('should sign a message with DER format', async () => {
     const key = await crypto.generateKey(algGenerate, true, ['sign']);
 
     const data = 'abcdefg';
     const alg = { name: 'EDDSA', namedCurve: 'ed25519', hash: { name: 'SHA-256' }, format: 'DER' };
```

Comment:
```suggestion
    const alg = { name: 'EdDSA', namedCurve: 'ed25519', hash: { name: 'SHA-256' }, format: 'DER' };
```

https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms